content/200-concepts/100-components/03-prisma-migrate/100-legacy-migrate.mdx

```diff
@@ -178,26 +178,26 @@ prisma migrate up --experimental
+
+| Operation                           | SQL                             |                                           Supported                                           |
+| :---------------------------------- | :------------------------------ | :-------------------------------------------------------------------------------------------: |
+| Create a new table                  | `CREATE TABLE`                  |                                              ✔️                                               |
+| Rename an existing table            | `ALTER TABLE` + `RENAME`        |                               Not yet (workaround coming soon)                                |
+| Delete an existing table            | `DROP TABLE`                    |                                              ✔️                                               |
+| Add a column to an existing table   | `ALTER TABLE` + `ADD COLUMN`    |                                              ✔️                                               |
+| Rename an existing column           | `ALTER TABLE` + `RENAME COLUMN` |                               Not yet (workaround coming soon)                                |
+| Delete an existing column           | `ALTER TABLE` + `DROP COLUMN`   |                                              ✔️                                               |
+| Set primary keys ([IDs]())          | `PRIMARY KEY`                   |                                              ✔️                                               |
+| Define [relations]() (foreign keys) | `FOREIGN KEY` + `REFERENCES`    |                                              ✔️                                               |
+| Make columns [optional/required]()  | `NOT NULL`                      |                                              ✔️                                               |
+| Set [unique constraints]()          | `UNIQUE`                        |                                              ✔️                                               |
+| Set [default]() values              | `DEFAULT`                       |                                              ✔️                                               |
+| Define [enums]()                    | `ENUM`                          |                                              ✔️                                               |
+| Create [indexes]()                  | `CREATE INDEX`                  |                                              ✔️                                               |
+| Cascading deletes                   | `ON DELETE`                     | Not yet ([workaround](/guides/database/advanced-database-tasks/cascading-deletes/postgresql)) |
```

Comment:
Unrelated: This is actually wrong now, isn't it?

Reply:
Right, yeah, we have cascading deletes now. can remove the flag and change the link

Reply:
Wait, this section is about legacy migrate, so from version 2.12.0 and earlier. This is valid.

Reply:
Oh you are right. Did not even notice that as I have never looked at the file/page.